From b62dd6b8798b110fa1b7d0df4fc4b9f30cea4e68 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Tue, 29 Dec 2020 09:31:57 +0000
Subject: [PATCH 66/79] arm64: dts: meson: fixes for WeTek common dtsi

Add GPIO bindings (copied from Nanopi-K2/Odroid-C2) to resolve an issue
with Ethernet probing in mainline u-boot. Also add eee-broken.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 arch/arm64/boot/dts/amlogic/meson-gxbb-wetek.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/amlogic/meson-gxbb-wetek.dtsi b/arch/arm64/boot/dts/amlogic/meson-gxbb-wetek.dtsi
index a350fee1264d..9c2f3c43ee67 100644
--- a/arch/arm64/boot/dts/amlogic/meson-gxbb-wetek.dtsi
+++ b/arch/arm64/boot/dts/amlogic/meson-gxbb-wetek.dtsi
@@ -6,6 +6,7 @@
  */
 
 #include "meson-gxbb.dtsi"
+#include <dt-bindings/gpio/gpio.h>
 
 / {
 	aliases {
@@ -145,6 +146,7 @@
 		eth_phy0: ethernet-phy@0 {
 			/* Realtek RTL8211F (0x001cc916) */
 			reg = <0>;
+			eee-broken-1000t;
 
 			reset-assert-us = <10000>;
 			reset-deassert-us = <80000>;
-- 
2.17.1

